example/weather/Program.cs

```diff
@@ -1,12 +1,11 @@
-﻿using System.Diagnostics;
-using Serilog;
+﻿using Serilog;
 using Serilog.Events;
 using Serilog.Templates.Themes;
 using SerilogTracing;
 using SerilogTracing.Expressions;
 using SerilogTracing.Sinks.OpenTelemetry;
 
-Log.Logger = new LoggerConfiguration()
+await using var logger = new LoggerConfiguration()
```

Comment:
Not initializing the static logger is fine, but it's not really the pit-of-success option since `Log.*` calls are then silently ignored. (This works for teams with a well-known no-static-logger convention, but for an introductory example, initializing `Log.Logger` is best.)

Reply:
I'll revert this example :+1: